menas/ui/service/GenericService.js

```diff
@@ -33,9 +33,40 @@ var GenericService = new function () {
     })
   };
 
+  this.runStatusFormatter = function(sStatus) {
+    if(sStatus === "failed") return "Failed";
+    else if(sStatus === "successful") return "Successfull";
```

Comment:
```suggestion
    else if(sStatus === "successful") return "Successful";
```

Reply:
Sigh at some stage we should probably start using i18n